modules/nw-ingress-operator-logs.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-operator-logs-{context}"]
+= View Ingress Operator logs
+
+You can view your Ingress Operator logs.
```

Comment:
Using "your" for the operator feels odd: the platform deploys and manages an instance of the operator.  This stands in contrast to ingress controllers, where  the user can create one or more `IngressController` resources, so "your ingress controller" feels less odd.  I realize this is pretty subjective, so feel free to disregard.

Reply:
That makes sense, thanks!